bamazonCustomer.js

```diff
@@ -0,0 +1,146 @@
+var inquirer = require("inquirer"); // User input from the terminal
+var mysql = require("mysql"); // MySQL database 
+require("console.table"); // Print Tables
+
+
+// Connection parameters for the MySQL database
+var connection = mysql.createConnection({
+	host: "127.0.0.1",
+	port: 3306,
+	user: "root",
+	password: "",
```

Comment:
It could make for a better experience for the user to include a process.env variable that gets assigned in the command line so that the user doesn't have to alter the file to make it work for their sql setup.